modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -25,16 +25,17 @@ $ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
 .. Edit the secret and add the base64 value to an `oidc.keycloak.clientSecret` key:
 +
 [source,terminal]
----- 
-$ oc edit secret openshift-gitops-secret -n <namespace>
+----
+$ oc edit secret argocd-secret -n <namespace>
 ----
 +
 .Example YAML of the secret
 [source,yaml]
 ----
-yaml apiVersion: v1 
-kind: Secret 
-metadata: name: argocd-secret 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-secret
 data:
   oidc.keycloak.clientSecret: ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …
```

Comment:
can we get rid of those 3 dots`(ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …)` in the end, after the encoded secret value?

Reply:
Sure thing!